src/parser/bison_parser.y

```diff
@@ -374,24 +383,65 @@ execute_statement:
 
 /******************************
  * Import Statement
+ * IMPORT FROM TBL FILE 'test/students.tbl' INTO students
+ * COPY students FROM 'test/students.tbl' (WITH FORMAT TBL)
```

Comment:
Did we already use parentheses for optional parts in this file? Otherwise, brackets [] would be more common.